grid/social-media/followers/maps/twitter.suma

```diff
@@ -77,3 +56,89 @@ operation MapFollower {
     imageUrl: follower.profile_image_url
   }
 }
+
+//Twitter rate limits: https://developer.twitter.com/en/docs/twitter-api/rate-limits
+operation MapRateLimit {
+  totalRequestsHeader = args.headers['x-rate-limit-limit']
+  remainingRequestsHeader = args.headers['x-rate-limit-remaining']
+  resetTimestampHeader = args.headers['x-rate-limit-reset']
+
+  totalRequests = undefined
+  remainingRequests = undefined
+  remainingRequestsPercentage = undefined
+  resetTimestamp = undefined
+
+  set if(totalRequestsHeader && parseInt(totalRequestsHeader, 10) !== 'NaN') {
+    totalRequests = parseInt(totalRequestsHeader, 10)
+  }
+
+  set if(remainingRequestsHeader && parseInt(remainingRequestsHeader, 10) !== 'NaN') {
+    remainingRequests = parseInt(remainingRequestsHeader, 10)
+  }
+
+  set if(totalRequests > 0 && remainingRequests !== undefined) {
+    remainingRequestsPercentage = remainingRequests / totalRequests * 100
+  }
+
+  set if(resetTimestampHeader && parseInt(resetTimestampHeader, 10) !== 'NaN') {
+    resetTimestamp = parseInt(resetTimestampHeader, 10)
+  }  
+
+  return {
+    bucket = 'get-users-followed-lists'
```

Comment:
Hmm, I don't quite follow the relation of this bucket name and the docs nor endpoint; how about just copying it verbatim from the docs as `Follows lookup`?

```suggestion
    bucket = 'Follows lookup'
```

(or if you really don't like spaces)

```suggestion
    bucket = 'follows-lookup'
```

Reply:
That was my mistake. I misinterpreted Twitter rate limit documentation. 👍